openslides_backend/http/views.py

```diff
@@ -89,11 +89,12 @@ def dispatch(self, body: RequestBody, headers: Headers) -> ResponseBody:
         payload: PresenterPayload = body
 
         # Handle request.
-        handler: Presenter = PresenterHandler()
+        handler: Presenter = PresenterHandler(
```

Comment:
Why not like for ActionHandler? If your way is better then we should change the way of action handler. Then please open an extra issue for this.

Reply:
> Then please open an extra issue for this.
See https://github.com/OpenSlides/openslides-backend/issues/43